### PR TITLE
portlet

### PR DESCRIPTION
**Pull request by brianlindsy**

### CODE REVIEW

File: src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+samplePortlet.hello=Hello {0}
+
+action.NEW_PERMISSION_FOR_SAMPLE=Caliphr Permission 1
```
Comment (andlar) [inline src/main/resources/content/Language.properties]: Probably should have a different permission

**Review by andlar — CHANGES_REQUESTED** [review]
Can you also add the checkstyle.xml from the other PULSE projects too?

**Review by andlar — APPROVED** [review]

**Comment by andlar** [comment]
Oh, is there a README? If not, can you put in a basic one with steps on build / deploy?

**Review by andlar — APPROVED** [review]

**Review by ssavarala — APPROVED** [review]

**Review by sriniadhiwork — APPROVED** [review]
small suggestion: GrantedPermissionModel extends GrantedAuthority, maybe we can annotate getAuthority with @Override for clarity. also the Boolean value is used for revokes I believe, perhaps naming as Boolean granted?

**Pull request merged**